indico/modules/events/timetable/templates/_timetable.html

```diff
@@ -0,0 +1,21 @@
+{% macro render_timetable(timetable_data, event_info, timetable_layout) %}
+    <div id="timetable" style="position: relative;">
+        <div class="timetablePreLoading" style="height: 300px;">
+            <div class="text" style="padding-top: 200px;">
+                {% trans %}Building timetable...{% endtrans %}
+            </div>
+        </div>
+        <div class="clearfix"></div>
+    </div>
+
+    <script>
+        $(window).on('load', function() {
+            var timetableLayout = {{ timetable_layout|tojson }} || null;
```

Comment:
Why not just default to `None` in the Python code? That'll be converted to `null` in tojson